pages/api/v1/answers.ts

```diff
@@ -785,7 +786,10 @@ export async function streamResponseToUser(
   let incompleteChunk = "";
   let first = true;
   // Below buffer is used to store the partial value of a variable if it's split across multiple chunks
-  let valueVariableBuffer = "";
+  let variableBuffer = "";
+  const variableValueMap = valueVariableMap
```

Comment:
i'm having a seizure with these variable names, can we call them something clearer like "mapIdentifiersToPlaceholders" and "mapPlaceholdersToIdentifiers"?

Reply:
I think generally the words "variable" and "value" are confusing, is the variable the original id or the replacement? I think identifier and placeholder probably work, thoughts?